src/CMakeLists.txt

```diff
@@ -69,12 +69,23 @@ ENDIF()
 ADD_LIBRARY(common_obj OBJECT ${common_src})
 SET_TARGET_PROPERTIES(common_obj PROPERTIES
     POSITION_INDEPENDENT_CODE ON
-    COMPILE_DEFINITIONS "MQTT_EXPORTS=1")
+	COMPILE_DEFINITIONS "MQTT_EXPORTS=1")
+TARGET_INCLUDE_DIRECTORIES(common_obj
+    PRIVATE ${CMAKE_BINARY_DIR}
+)
 
 ADD_EXECUTABLE(MQTTVersion MQTTVersion.c)
+TARGET_INCLUDE_DIRECTORIES(MQTTVersion
+    PRIVATE ${CMAKE_BINARY_DIR}
+)
 
-ADD_LIBRARY(paho-mqtt3c SHARED $<TARGET_OBJECTS:common_obj> MQTTClient.c)
-ADD_LIBRARY(paho-mqtt3a SHARED $<TARGET_OBJECTS:common_obj> MQTTAsync.c)
+IF (PAHO_BUILD_STATIC)
```

Comment:
I think you removed your definition of the `PAHO_BUILD_STATIC` cache variable.

Reply:
I don't believe so... I did start over from my first submission back to the develop branch. I'm sorry if that made things confusing.

Reply:
I removed the `PAHO_BUILD_SHARED` that I had added. Now, if the previously existing `PAHO_BUILD_STATIC` is set, the library will only build the static libraries _and not the shared_ (new behaviour). If it is not set, the shared will build and not the static -- except for the import libraries on Windows (existing behaviour).